pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -274,7 +274,8 @@ func computeActiveCondition(ctx context.Context, pa *pav1alpha1.PodAutoscaler, p
 
 // activeThreshold returns the scale required for the pa to be marked Active
 func activeThreshold(ctx context.Context, pa *pav1alpha1.PodAutoscaler) int {
-	min, _ := pa.ScaleBounds()
+	asConfig := config.FromContext(ctx).Autoscaler
+	min, _ := pa.ScaleBounds(asConfig)
 	if !pa.Status.IsScaleTargetInitialized() {
 		initialScale := resources.GetInitialScale(config.FromContext(ctx).Autoscaler, pa)
```

Comment:
looks like you could re-use `asConfig` here now

Reply:
ugh that was my intent, guess i missed it. good catch.